content/api/authentication/enterpriseuser/_index.nb.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Autentisere med virksomhetsbruker
+linktitle: Virksomhetsbruker
+description: Beskrivelse hvordan man kan benytte virksomhetsbrukere i Altinn 3
+toc: true
+weight: 100
+--
+
+## Overordne beskrivelse
+
+Virksomhetsbrukere er et konsept hvor man kan benytte et virksomhetssertifikat i kombinasjon med et brukernavn/passord.
+
+Dette gir brukere som er knyttet mot en virksomhet som kan få delegert roller for tilgangsstyring. 
+
+## Virksomhetsbrukere i Altinn 3
+
+Virksomhetbrukere i Altinn 3 kan benyttes via vanlig virksomhetsbrukerinnlogging på Altinn.no med sertifikat lagret i nettleser, men er mest relevant i sammenheng med API bruk.
+
+Dette gjøres ved å kombinere maskinporten for selve autentisering av sertifikatet, samt en innveksling av maskinportentoken sammen med brukernavn/passord.
```

Comment:
Ordet _kombinere_ virker feil. Dette er en 2-stegsprosess. Kan forklares bedre.

Reply:
Når jeg tenker meg om blir det vel feil å snakke om brukernavn og passord i denne sammeheng. 2-steg prosessen jeg tenkte på er 1. Autentisering i Maskinporten 2. Innveksling i Altinn 3 token.

Reply:
Men for virksomhetsbrukere må man legge ved brukernavn/passord når man veksler inn